src/components/pages/Login/index.tsx

```diff
@@ -46,8 +46,8 @@ export default function Login(): JSX.Element {
   return (
     <Layout>
       <Content className={styles.content}>
```

Comment:
Im not sure if its cause by styling bug here or somewhere else in this file / the CSS module, but when the CustomAlert component pops up on a failed login or other error, it moves everything else around on the page. It should just appear and not effect anything else